modules/cluster/main.tf

```
@@ -79,7 +79,7 @@ resource "aws_ecs_cluster" "this" {
   }
 
   dynamic "setting" {
-    for_each = [var.cluster_settings]
+    for_each = var.cluster_settings
```

```suggestion
    for_each = flatten([var.cluster_settings])
```

This will be equal to `list(map(string))` for both types: `list(map(string))` and `map(string)`.

---

@antonbabenko thanks for suggestions!
I'll prepare the appropriate changes.